src/main/java/Timo/Timo.java

```diff
@@ -0,0 +1,584 @@
+package Timo;
+
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.DateTimeException;
+import java.time.LocalDateTime;
+import java.util.Scanner;
+import java.util.*;
+import java.io.File;
+import java.time.format.DateTimeFormatter;
+
+class Task {
+    private String taskype = "T";
+    private boolean mark;
+    private String val;
+    public Task(boolean mark, String val) {
```

Comment:
Perhaps something more descriptive could be used in place of ```val```?

Reply:
Ok that's a good idea, thanks!